Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -170,7 +171,9 @@ - (void)setup {
     self.gregorian = [[NSCalendar alloc] initWithCalendarIdentifier:NSCalendarIdentifierGregorian];
 
     self.calendarView = [[FSCalendar alloc] initWithFrame:CGRectZero];
-    if (self.configuration.rowHeight != nil) {
+    if (self.configuration.rowHeight == nil) {
+        self.calendarView.rowHeight = BPKCalendarDefaultCellHeight;
```

Comment:
The default height in FSCalendar no longer works for us as we're making the cells bigger. As a result here I'm applying our own default height.